### PR TITLE
docs: add tutorials preference and clean

### PR DESCRIPTION
**Pull request by sdiazlor**

### CODE REVIEW

**Comment by sdiazlor** [comment]
@davidberenstein1957 Do you know for zenml if this is the documentation reference or there was another? https://github.com/zenml-io/zenml-projects/blob/main/llm-complete-guide/notebooks/argilla_embeddings.ipynb

**Comment by davidberenstein1957** [comment]
@sdiazlor , it was this one https://docs.zenml.io/user-guide/llmops-guide/finetuning-embeddings

File: docs/sections/pipeline_samples/index.md [hunk 1]
```diff
@@ -11,13 +11,31 @@ hide: toc
...
+-   __Generate a preference dataset__
+
+    ---
+
+    Learn about synthetic data generation for ORPO and DPO.
+
+    [:octicons-arrow-right-24: Tutorial](tutorials/generate_preference_dataset.ipynb)
+
+
+-   __Clean an existing dataset__
```
Comment (davidberenstein1957) [inline docs/sections/pipeline_samples/index.md:23]: preference dataset right?

File: docs/sections/pipeline_samples/tutorials/clean_existing_dataset.ipynb [hunk 2]
```diff
@@ -0,0 +1,473 @@
+{
```
Comment (davidberenstein1957) [inline docs/sections/pipeline_samples/tutorials/clean_existing_dataset.ipynb:1]: <p>can you add this to my tutorial too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (davidberenstein1957) [inline docs/sections/pipeline_samples/tutorials/clean_existing_dataset.ipynb:1]: <p>perhaps we can add an embedded dataset viewer here too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (davidberenstein1957) [inline docs/sections/pipeline_samples/tutorials/clean_existing_dataset.ipynb:1]: <p>Can you add the custom step to the intro too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (davidberenstein1957) [inline docs/sections/pipeline_samples/tutorials/clean_existing_dataset.ipynb:1]: <p>should we add the <code>process</code> function, perhaps t can show what happens to and input example for each step? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (davidberenstein1957) [inline docs/sections/pipeline_samples/tutorials/clean_existing_dataset.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #33.        load_dataset &gt;&gt; evaluate_responses &gt;&gt; [keep_columns, to_argilla]
</pre><p>does this work? I would say we first filter out the columns? or do we also do that within the to_argilla Step?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

File: docs/sections/pipeline_samples/tutorials/generate_preference_dataset.ipynb [hunk 3]
```diff
@@ -0,0 +1,427 @@
+{
```
Comment (davidberenstein1957) [inline docs/sections/pipeline_samples/tutorials/generate_preference_dataset.ipynb:1]: <p>where are we using torch?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (davidberenstein1957) [inline docs/sections/pipeline_samples/tutorials/generate_preference_dataset.ipynb:1]: <p>can you also make sure this align with my tutorial?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (davidberenstein1957) [inline docs/sections/pipeline_samples/tutorials/generate_preference_dataset.ipynb:1]: <p>Maybe we can just use headers for the names like Load the dataset and Generate responses</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (davidberenstein1957) [inline docs/sections/pipeline_samples/tutorials/generate_preference_dataset.ipynb:1]: <p>here the bullet points are off, perhaps some spacing or missing enters?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (davidberenstein1957) [inline docs/sections/pipeline_samples/tutorials/generate_preference_dataset.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #46.        load_dataset &gt;&gt; generate_responses &gt;&gt; group_responses &gt;&gt; evaluate_responses &gt;&gt; [format_dpo, to_argilla]
</pre><p>again, should we do them sequential, or do you want to show to [] opearation along with the &gt;&gt;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (davidberenstein1957) [inline docs/sections/pipeline_samples/tutorials/generate_preference_dataset.ipynb:1]: <p>can we add this to distilabel-internal-testing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (sdiazlor) [reply docs/sections/pipeline_samples/tutorials/generate_preference_dataset.ipynb:1]: <p>I used a clean environment, and asked me to install it to run something in the pipeline</p>

File: docs/sections/pipeline_samples/tutorials/clean_existing_dataset.ipynb (hunk 2 above)
Comment (sdiazlor) [reply docs/sections/pipeline_samples/tutorials/clean_existing_dataset.ipynb:1]: <p>Yes, I'll change it for this one</p>

File: docs/sections/pipeline_samples/tutorials/generate_preference_dataset.ipynb (hunk 3 above)
Comment (sdiazlor) [reply docs/sections/pipeline_samples/tutorials/generate_preference_dataset.ipynb:1]: <p>In this case, I prefer to show [] to show the difference, as Format already shows the chosen/rejected automatically, but with argilla is manually</p>

Comment (davidberenstein1957) [reply docs/sections/pipeline_samples/tutorials/generate_preference_dataset.ipynb:1]: @gabrielmbmb is this intended? I would assume we would not need this when just working with InferenceEndpoints.

**Review by davidberenstein1957 — APPROVED** [review]

**Pull request merged**